packages/webpack-config/src/index.js

```diff
@@ -72,6 +78,23 @@ const baseConfig = {
 				parser: { requireEnsure: false },
 			},
 
+			// JavaScript - babel-loader transpiles our javascript to ensure browser compatability
+			{
+				test: /\.(js|ts)$/,
```

Comment:
Could we move the Typescript change in a separate PR to keep this one about the file loader? There are a few things we would want to consider with Typescript, like, do we add it to the base config, or an optional extension config? Do we run TS files through ts-loader prior to babel-loader, like create react app and vue-cli?